app/src/routes/onboarding/forms/user.ts

```diff
@@ -0,0 +1,254 @@
+	const { t } = useI18n();
+	return [
+		{
+			collection: 'onboarding',
+			name: t('fields.directus_users.first_name'),
+			field: 'first_name',
+			type: 'string',
+			schema: {
+				name: 'first_name',
+				table: 'onboarding',
+				schema: 'public',
+				data_type: 'character varying',
+				is_nullable: false,
+				generation_expression: null,
+				default_value: null,
+				is_generated: false,
+				max_length: 255,
+				comment: null,
+				numeric_precision: null,
+				numeric_scale: null,
+				is_unique: false,
+				is_primary_key: false,
+				has_auto_increment: false,
+				foreign_key_schema: null,
+				foreign_key_table: null,
+				foreign_key_column: null,
+			},
+			meta: {
+				id: 17,
+				collection: 'onboarding',
+				field: 'first_name',
+				special: null,
+				interface: 'input',
+				options: { placeholder: t('fields.directus_users.first_name'), trim: true },
+				display: null,
+				display_options: null,
+				readonly: false,
+				hidden: false,
+				sort: 2,
+				width: 'half',
+				translations: null,
+				note: null,
+				conditions: null,
+				required: false,
+				group: null,
+				validation: null,
+				validation_message: null,
+			},
+		},
+		{
+			collection: 'onboarding',
+			name: t('fields.directus_users.last_name'),
+			field: 'last_name',
+			type: 'string',
+			schema: {
+				name: 'last_name',
+				table: 'onboarding',
+				schema: 'public',
+				data_type: 'character varying',
+				is_nullable: false,
+				generation_expression: null,
+				default_value: null,
+				is_generated: false,
+				max_length: 255,
+				comment: null,
+				numeric_precision: null,
+				numeric_scale: null,
+				is_unique: false,
+				is_primary_key: false,
+				has_auto_increment: false,
+				foreign_key_schema: null,
+				foreign_key_table: null,
+				foreign_key_column: null,
+			},
+			meta: {
+				id: 18,
+				collection: 'onboarding',
+				field: 'last_name',
+				special: null,
+				interface: 'input',
+				options: { placeholder: t('fields.directus_users.last_name'), trim: true },
+				display: null,
+				display_options: null,
+				readonly: false,
+				hidden: false,
+				sort: 3,
+				width: 'half',
+				translations: null,
+				note: null,
+				conditions: null,
+				required: false,
+				group: null,
+				validation: null,
+				validation_message: null,
+			},
+		},
+		{
+			collection: 'onboarding',
+			name: t('fields.directus_users.email'),
+			field: 'email',
+			type: 'string',
+			schema: {
+				name: 'email',
+				table: 'onboarding',
+				schema: 'public',
+				data_type: 'character varying',
+				is_nullable: false,
+				generation_expression: null,
+				default_value: null,
+				is_generated: false,
+				max_length: 255,
+				comment: null,
+				numeric_precision: null,
+				numeric_scale: null,
+				is_unique: false,
+				is_primary_key: false,
+				has_auto_increment: false,
+				foreign_key_schema: null,
+				foreign_key_table: null,
+				foreign_key_column: null,
+			},
```

Comment:
Can we somehow validate it actually is an email here or we'll need to show the errors same as the Users form. Getting an "unknown error" for invalid emails is not the best first experience.
![firefox_f4vT38UQ2d](https://github.com/directus/directus/assets/9389634/ce2e4ffd-fe58-47ea-877a-1ce651976fcb)

Reply:
I changed it now to either show api errors or the unknown error but I think an even better solution would be to user our translated error messages otherwise the error will always be in english I think?

![image](https://github.com/directus/directus/assets/14810858/cac1df3e-c440-460c-9fe1-863440cd7e64)

Reply:
Resolved via https://github.com/directus/directus/pull/19837/commits/64b66cf30ee31a5f4e631ed824c42f768dcfc6b7